src/anim/evaluator/anim-track.js

```diff
@@ -39,6 +41,14 @@ class AnimTrack {
         return this._curves;
     }
 
+    get events() {
+        return this._animEvents?.events || [];
```

Comment:
when value is undefined or null, it's better to return that instead of creating it at that point. Alternatively, initialize it to [] in the constructor (default value for the parameter)